scripts/validate_docstrings.py

```diff
@@ -163,10 +163,12 @@ def double_blank_lines(self):
 
     @property
     def summary(self):
-        if not self.doc['Extended Summary'] and len(self.doc['Summary']) > 1:
-            return ''
         return ' '.join(self.doc['Summary'])
 
+    @property
+    def num_summary_lines(self):
```

Comment:
Overall approach is much cleaner here, though I'm maybe -0.5 on the property as I don't see this really being used outside of the one check you have, in which case you can probably just do `len(self.doc['Summary'])` directly

I'll defer to the @datapythonista on this one though